chain/rosetta-rpc/src/lib.rs

```diff
@@ -56,7 +57,10 @@ async fn check_network_identifier(
     }
 
     let status = client_addr
-        .send(near_client::Status { is_health_check: false, detailed: false }.with_span_context())
+        .send(
+            SpanWrappedMsg::from(near_client::Status { is_health_check: false, detailed: false })
+                .with_span_context(),
```

Comment:
It would be great if we could avoid having two related concepts in the same exact place – `with_span_context` already does some work related to enabling tracing of messages, so it seems pretty awkward to add another orthogonal concept here. Is there any reason these couldn't have been unified together?